tools/Microsoft.AspNet.Server.Kestrel.GeneratedCode/KnownHeaders.cs

```diff
@@ -190,6 +178,28 @@ public partial class {loop.ClassName}
         {Each(loop.Headers, header => @"
         private StringValues _" + header.Identifier + ";")}
 
+        {Each(loop.Headers, header => $@"
+        public StringValues Header{header.Identifier}
+        {{
+            get
+            {{
+                if ({header.TestBit()})
```

Comment:
Since `ClearFast` now actually sets all the properties to `StringValues.Empty`, I think we can remove the if/else and just `return _{header.Identifier};` in the getter.